src/xrc/xh_menu.cpp

```diff
@@ -101,6 +114,17 @@ wxObject *wxMenuXmlHandler::DoCreateResource()
             wxMenuItem *mitem = new wxMenuItem(p_menu, id, label,
                                                GetText(wxT("help")), kind);
 #if wxUSE_ACCEL
+            if (!extra_accels.empty())
+            {
+                const int entries_size = extra_accels.size();
+                for (wxString extra_accel : extra_accels)
+                {
+                    wxAcceleratorEntry entry;
+                    if (entry.FromString(extra_accel))
```

Comment:
Should we give an error if this fails? Or is this never supposed to fail here? If it isn't, maybe we should store `wxAcceleratorEntries` directly, i.e. `wxVector<wxAcceleratorEntry>`?